src/platform/forms-system/src/js/helpers.js

```diff
@@ -298,45 +298,83 @@ function isPartialAddress(data) {
   );
 }
 
+/**
+ * Replaces double-quotes with single.
+ * Replaces non-escaping backslashes with forward-slashes.
+ *
+ * @param {string} string
+ * @returns {string}
+ */
+export const replaceEscapedCharacters = string => {
+  return string
+    .replaceAll('"', "'")
+    .replace(/(?:\r\n|\n\n|\r|\n)/g, '; ')
+    .replace(/(?:\t|\f|\b)/g, '')
+    .replace(/\\(?!(f|n|r|t|[u,U][\d,a-fA-F]{4}))/gm, '/');
+};
+
 /**
  * Create a replacer function for JSON.stringify
  * A replacer function is the second argument to JSON.stringify
  *
- * @param {{
- *  allowPartialAddress?: boolean,
- * }} [options] - Options for the replacer
+ * @param {ReplacerOptions} [options] - Options for the replacer
+ * @returns {(key, value) => any} The replacer function
  */
 export function createStringifyFormReplacer(options) {
+  /**
+   * The replacer function
+   *
+   * @param {*} key
+   * @param {*} value
+   * @returns {*} Replaced value
+   */
   const replacerFn = (key, value) => {
-    if (!options?.allowPartialAddress && isPartialAddress(value)) {
-      return undefined;
-    }
+    /**
+     * Replaces objects (including arrays) with other values
+     *
+     * @param {Object} object
+     * @returns {*} The replacement value
+     */
+    const replaceObject = object => {
+      // Clean up empty objects in arrays
+      if (Array.isArray(value)) {
+        const newValues = value.filter(v => !!replacerFn(key, v));
```

Comment:
**QUESTION/CONSIDER CHANGING**
It looks to me like this closure is not intentional. I think `value` should be `object` here. Is that correct, or is this intentional?

Reply:
Good catch! Fixed and added a test to `helpers.unit.spec.js` to make sure that aspect of `createStringifyFormReplacer` is working correctly.